components/rating/rating.html

```diff
@@ -0,0 +1,114 @@
+                    use?</a></li>
+            <div class="sidebar-heading heading-3">Components</div>
+            <li class="sidebar-item"><a href="../../components/alerts/alert.html" class="sidebar-link">Alert</a></li>
+            <li class="sidebar-item"><a href="../../components/avatar/avatar.html" class="sidebar-link">Avatar</a></li>
+            <li class="sidebar-item"><a href="../../components/badge/badge.html" class="sidebar-link">Badge</a></li>
+            <li class="sidebar-item"><a href="../../components/button/button.html" class="sidebar-link">Button</a></li>
+            <li class="sidebar-item"><a href="../../components/card/card.html" class="sidebar-link">Card</a></li>
+            <li class="sidebar-item"><a href="../../components/image/image.html" class="sidebar-link">Image</a></li>
+            <li class="sidebar-item"><a href="../../components/input/input.html" class="sidebar-link">Input</a></li>
+            <li class="sidebar-item"><a href="../../components/lists/lists.html" class="sidebar-link">Lists</a></li>
+            <li class="sidebar-item"><a href="../../components/modal/modal.html" class="sidebar-link">Modal</a></li>
+            <li class="sidebar-item"><a href="../../components/navigation/navigation.html"
+                    class="sidebar-link">Navigation</a></li>
+            <li class="sidebar-item"><a href="../../components/rating/rating.html" class="sidebar-link">Rating</a></li>
+            <li class="sidebar-item"><a href="../../components/text-utilities/text-utilities.html"
+                    class="sidebar-link">Text-utilities</a></li>
+        </ul>
+    </nav>
+
+    <main class="main-container">
+        <h2 class="heading-2 fw-regular m-tp-one">Rating</h2>
+        <hr>
+        <section class="section">
+            <p class="heading-4 m-tp-one fw-lght">To make star rating, copy paste the code provided.</p>
+            <div class="wrapper p-1 m-tp-one pos-rltv min-hght-20">
+                <div class="rating-star-container">
+                    <i class="rating-star far fa-star"></i>
+                    <i class="rating-star far fa-star"></i>
+                    <i class="rating-star far fa-star"></i>
+                    <i class="rating-star far fa-star"></i>
+                    <i class="rating-star far fa-star"></i>
+                </div>
+            </div>
+        </section>
+
+        <pre>
+            <code class="language-markup">
+                <div class="rating-star-container">
+                    <i class="rating-star far fa-star"></i>
+                    <i class="rating-star far fa-star"></i>
+                    <i class="rating-star far fa-star"></i>
+                    <i class="rating-star far fa-star"></i>
+                    <i class="rating-star far fa-star"></i>
+                </div>
+
+                <!-- code used for changing rating dynamically: -->
```

Comment:
try to remove these comments as it takes extra space

Reply:
It is written so the user knows what does below the code do..